org.eclipse.winery.repository.ui/src/app/wineryNamespaceSelector/wineryNamespaceSelector.component.ts

```diff
@@ -121,4 +135,18 @@ export class WineryNamespaceSelectorComponent implements OnInit, ControlValueAcc
     registerOnTouched(fn: any) {
         this.onTouchedCallback = fn;
     }
+
+    applyNamespace() {
+        localStorage.setItem(StartNamespace.LocalStorageEntry.toString(), this.initNamespaceString);
+        this.innerValue = this.initNamespaceString !== '' ? this.applyToscaTypeToNamespace(this.initNamespaceString) : '';
+    }
+
+    /**
+     * Returns namespace with tosca type
+     * @returns {string} namepspace which ends with /<toscaType>
+     */
+    private applyToscaTypeToNamespace(namespace: string) {
```

Comment:
Why isn't the parameter called `namespaceStart` - to be consistent with the variable?